### PR TITLE
Add MangaPark v3

### PR DESCRIPTION
**Pull request by FourTOne5**
So yeah I acccidentally deleted the previous PR.

For the moment I am keeping both MangaPark v2 and v3 as v3 is still in beta.

While v2 is SingleLang v3 is MultiLang (Although the site for the moment is still English only.)

I am also pretty new Kotlin so give me some feedback. And I hope that the veterans will clean up the code. Though the code is pretty function able it will need some rework on the code in Line 274 - 284 (which I am dumb to do.)

### CODE REVIEW

File: src/en/mangapark/build.gradle [hunk 1]
```diff
@@ -2,10 +2,10 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 ext {
-    extName = 'MangaPark'
+    extName = 'MangaPark v2'
```
Comment (CarlosEsco) [inline src/en/mangapark/build.gradle:5]: I believe if you change this time you are going to cause issues

Comment (FourTOne5) [reply src/en/mangapark/build.gradle:5]: Ohh yeah missing download.. removing it in a bit.

File: src/all/mangapark/src/eu/kanade/tachiyomi/extension/all/mangapark/MangaPark.kt [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+        viewer.select("div.item").forEach() { i ->
+            val index = i.select("span").text().split("/".toRegex())[0].toInt() - 1
+            val imageUrl = i.select("img").attr("abs:src")
+            pages.add(Page(index, "", imageUrl))
+        }
+
+        return pages
+    }
+
+    private val cryptoJS by lazy {
```
Comment (CarlosEsco) [inline src/all/mangapark/src/eu/kanade/tachiyomi/extension/all/mangapark/MangaPark.kt:286]: maybe my browser search isn't working but is this being used?

**Comment by FourTOne5** [comment]
@CarlosEsco Check now.

File: src/all/mangapark/src/eu/kanade/tachiyomi/extension/all/mangapark/MangaPark.kt (hunk 2 above)
Comment (FourTOne5) [reply src/all/mangapark/src/eu/kanade/tachiyomi/extension/all/mangapark/MangaPark.kt:286]: Well this was made with batoto as base so there are some junk code

Comment (CarlosEsco) [reply src/all/mangapark/src/eu/kanade/tachiyomi/extension/all/mangapark/MangaPark.kt:286]: can you remove anything not used

Comment (FourTOne5) [reply src/all/mangapark/src/eu/kanade/tachiyomi/extension/all/mangapark/MangaPark.kt]: I rechecked the code and I think this was last of the junk code.

**Pull request merged**